openpype/hosts/maya/plugins/publish/extract_fbx_animation.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+import os
+
+from maya import cmds  # noqa
+import pyblish.api
+
+from openpype.pipeline import publish
+from openpype.hosts.maya.api import fbx
+from openpype.hosts.maya.api.lib import namespaced
+
+
+class ExtractFBXAnimation(publish.Extractor):
+    """Extract Rig in FBX format from Maya.
+
+    This extracts the rig in fbx with the constraints
+    and referenced asset content included.
+    This also optionally extract animated rig in fbx with
+    geometries included.
+
+    """
+    order = pyblish.api.ExtractorOrder
+    label = "Extract Animation (FBX)"
+    hosts = ["maya"]
+    families = ["animation.fbx"]
+
+    def process(self, instance):
+        # Define output path
+        staging_dir = self.staging_dir(instance)
+        filename = "{0}.fbx".format(instance.name)
+        path = os.path.join(staging_dir, filename)
+        path = path.replace("\\", "/")
+
+        fbx_exporter = fbx.FBXExtractor(log=self.log)
+        out_set = instance.data.get("animated_skeleton", [])
+        # Export
+        instance.data["constraints"] = True
+        instance.data["skeletonDefinitions"] = True
+        instance.data["referencedAssetsContent"] = True
+
+        fbx_exporter.set_options_from_instance(instance)
+
+        out_set_name = next(out for out in out_set)
+        # Export from the rig's namespace so that the exported
+        # FBX does not include the namespace but preserves the node
+        # names as existing in the rig workfile
+        namespace, relative_out_set = out_set_name.split(":", 1)
+        with namespaced(
+            ":" + namespace,
+            new=False,
+            relative_names=True
+        ) as namespace:
+            fbx_exporter.export(relative_out_set, path)
```

Comment:
A question - why do we need to set `relative_names=True`? Is that required for the FBX exporter to ignore the namespace?
Because if not, we could just set `relative_names=`False` and export with `out_set_name` instead of the relative name? :)

Reply:
We need to import the fbx animation into other dcc such as Unreal and some of them require the strict hierarchy names so that's why we need to ignore the namespace for this extractor by default. 
but we can do a setting for this and enabled by default. Once the users dont want to have the namespace, they can just disable it.

Reply:
I understand - what I meant was; that if you were to go into the namespace with relative names `False` does it still export without the namespace. Did a quick test run on my end with export FBX and it does seem to require the relative names to be enabled - so will mark this resolved.